src/main/java/smile/identity/core/models/UserData.java

```diff
@@ -1,18 +1,24 @@
 package smile.identity.core.models;
 
+import lombok.AllArgsConstructor;
 import lombok.Value;
 
 @Value
+@AllArgsConstructor
 public class UserData {
-    boolean isVerifiedProcess = false;
+    boolean isVerifiedProcess;
     String name;
     String fbUserID;
-    String firstName = "Bill";
+    String firstName;
     String lastName;
     String gender;
     String email;
     String phone;
-    String countryCode = "+";
+    String countryCode;
     String countryName;
 
+    public UserData(){
+        this(false, "", "", "", "Bill", "", "", "", "+", "");
```

Comment:
Intrigued why this is Bill and not an empty string like the other values? 🤣

Reply:
Great question, that I do not have a great answer for sadly. I saw it was that way previously, and also the same in the ruby code and was afraid that if I changed it, it would mess something up that I wasn't aware about in the backend (lol). But just looked at the other SDKs and seems its safe to leave it as an empty string 😅 @beastawakens

Reply:
I can imagine it's just some leftover of dummy data 😆